addons/l10n_in/tests/test_l10n_in_fiscal_position.py

```diff
@@ -132,3 +132,87 @@ def test_l10n_in_fiscal_in_bill_to_ship_to(self):
             out_invoice.fiscal_position_id,
             self.env['account.chart.template'].ref('fiscal_position_in_export_sez_in')
         )
+
+    def test_l10n_in_fiscal_in_vendor_bills(self):
+        '''
+        In Purchase Document: Compare place of supply with vendor state
+        '''
+
+        self.env.company = self.default_company
+        template = self.env['account.chart.template']
+        company_state = self.env.company.state_id
+        other_state = self.env['res.country.state'].search([
+            ('id', '!=', company_state.id),
+            ('country_id', '=', company_state.country_id.id)],
+            limit=1
+        )
```

Comment:
Opening and closing brackets at the same indentation level greatly improves readability

```suggestion
        other_state = self.env['res.country.state'].search([
            ('id', '!=', company_state.id),
            ('country_id', '=', company_state.country_id.id)
        ], limit=1)
```